study-builder/studies/testboston/study-pdfs.conf

```diff
@@ -1,4 +1,87 @@
 {
-  # TODO: this needs work
-  "pdfs": []
+  "pdfs": [
+    {
+      "name": "testboston-consent",
+      "filename": "testboston-consent",
+      "displayName": "TestBoston consent pdf",
+      "mapping": {
+        "type": "CONSENT"
+      },
+      "versions": [
+        {
+          "tag": "v1",
+          "start": null,
+          "end": null,
+          "sources": [
+            {
+              "type": "ACTIVITY",
+              "activityCode": ${id.act.consent},
+              "versionTag": "v1"
+            }
+          ],
+          "files": [
+            {
+              "filepath": "pdfs/consent.pdf",
```

Comment:
The `consent.pdf` file is a dummy file I created as a placholder for now. I'm still waiting on the actual pdf.